src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveScriptEnvironmentServiceFactory.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Editor.Implementation.Interactive;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Scripting;
+
+namespace Microsoft.CodeAnalysis.Editor.Interactive
+{
+    [ExportWorkspaceServiceFactory(typeof(IScriptEnvironmentService), WorkspaceKind.Interactive), Shared]
+    internal sealed class InteractiveScriptEnvironmentServiceFactory : IWorkspaceServiceFactory
+    {
+        [Export(typeof(IScriptEnvironmentService)), Shared]
```

Comment:
Is this attribute needed? I wouldn't expect it to work since MEF wouldn't be able to construct an instance since there isn't a public, parameterless constructor or an `[ImportingConstructor]`

Reply:
Probably not. Will remove it.